### PR TITLE
Add windows build

### LINKED ISSUE #8: Please provide Windows builds
This feedstock is missing Windows builds.
@tbeu would you be kind enough and take a look at this please?

### PR DESCRIPTION
**Pull request by AliKhoda**
Fixes #8

### CODE REVIEW

File: recipe/bld.bat [hunk 1]
```diff
@@ -0,0 +1,2 @@
+appveyor DownloadFile https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-1.10/hdf5-1.10.0/bin/windows/extra/hdf5-1.10.0-win"%ARCH%"-VS2015-shared.zip
+7z x -y hdf5-1.8.13-win%ARCH%-VS2010-shared.zip
```
Comment (183amir) [inline recipe/bld.bat]: You should not download hdf5 here. It will come from conda.

**Comment by 183amir** [comment]
@conda-forge-admin, please rerender

**Comment by 183amir** [comment]
@tbeu, @AliKhoda and I are trying to build this on Windows. If you could help, I would really appreciate it.

**Comment by 183amir** [comment]
I cannot compile it using vs2088. See: https://ci.appveyor.com/project/183amir/libmatio-feedstock

**Comment by AliKhoda** [comment]
@conda-forge-admin, please rerender

**Comment by 183amir** [comment]
@conda-forge-admin, please rerender

**Comment by 183amir** [comment]
looks like travis takes 3 days to build things. Since this pull request is changing the windows builds, I will merge this.

**Comment by massich** [comment]
> @tbeu, @AliKhoda and I are trying to build this on Windows. If you could help, I would really appreciate it.

I think I came late to the party. But I'm happy to help. The windows builds are missing `matio.h` and `matio_pubconf.h` I'll open an issue.

**Comment by tbeu** [comment]
Why still matio v1.5.6?

**Comment by 183amir** [comment]
no reason. That needs to be updated.

**Comment by AliKhoda** [comment]
A pull request to fix version and missing headers has been made.

**Pull request merged**